src/pages/home/uploadSong/UploadSong.tsx

```diff
@@ -177,13 +180,11 @@ const UploadSong: FunctionComponent = () => {
           rootPath="home/upload-song"
           isProgressStepperVisible={ true }
           enableReinitialize={ true }
-          validateOnBlur={ false }
           routes={ [
             {
               element: <BasicSongDetails />,
               path: "",
               progressStepTitle: "Basic details",
-              navigateOnSubmitStep: false,
```

Comment:
Currently navigating on Upload Only (No Mint) after progress bar to `advanced-details` instead of 'Library' page. I believe that is why this was set as false and the `navigate("advanced-details");` was in the conditional above at line 96